lua/nvim-tree/renderer/components/floating-node.lua

```diff
@@ -0,0 +1,82 @@
+local M = {}
+
+local api = vim.api
+local fn = vim.fn
+
+local function hide(win)
+  if win then
+    if api.nvim_win_is_valid(win) then
+      api.nvim_win_close(win, true)
+    end
+  end
+end
+
+local popup_win = nil
+local function show()
+  hide(popup_win)
```

Comment:
Is this necessary?
The first autocommand should do the job.

Reply:
Yes, this is not necessary, I will remove it.